### ISSUE TITLE
Build Failing

### ISSUE THREAD

**Issue by fergusmeiklejohn** [MangelMaxime/Fulma#138]
I'm getting two build failures with this template:

(1) On a new project, in a clean folder -> I create the template -> `chmod a+x fake.sh` -> `./fake.sh build -t Watch` 

The Build fails with:

```
Restoring packages for /Users/fergus/Code/FableTest/LearnFable/build.proj...
  Generating MSBuild file /Users/fergus/Code/FableTest/LearnFable/obj/build.proj.nuget.g.props.
  Generating MSBuild file /Users/fergus/Code/FableTest/LearnFable/obj/build.proj.nuget.g.targets.
  Restore completed in 170.78 ms for /Users/fergus/Code/FableTest/LearnFable/build.proj.
No executable found matching command "dotnet-fake"
```

(2) On a existing project that was building fine yesterday -> `./fake.sh build -t Watch`

The build fails with:

```
darwin17
  Restore completed in 15.79 ms for /Users/fergus/Code/platform_one/platformOne/build.proj.
  Restore completed in 53.31 ms for /Users/fergus/Code/platform_one/platformOne/build.proj.
  Restore completed in 53.26 ms for /Users/fergus/Code/platform_one/platformOne/build.proj.
run Watch
Building project with version: LocalBuild
Shortened DependencyGraph for Target Watch:
<== Watch
   <== YarnInstall
      <== Install
         <== Clean

The resulting target order is:
 - Clean
 - Install
 - YarnInstall
 - Watch
Starting target 'Clean': NoDescription
  NoDescription
Finished 'Clean' in 00:00:00.0388650
Starting target 'Install': NoDescription
  NoDescription
Starting task 'DotNet:restore': platformOne.sln
/Library/Frameworks/Mono.framework/Versions/Current/Commands/mono --version
/usr/local/share/dotnet/dotnet  restore  platformOne.sln
  Performance:
   - Runtime: 56 milliseconds
  Performance:
   - Runtime: 52 milliseconds
  Restoring packages for /Users/fergus/Code/platform_one/platformOne/src/platformOne.fsproj...
  Generating MSBuild file /Users/fergus/Code/platform_one/platformOne/src/obj/platformOne.fsproj.nuget.g.props.
  Generating MSBuild file /Users/fergus/Code/platform_one/platformOne/src/obj/platformOne.fsproj.nuget.g.targets.
  Restore completed in 177.39 ms for /Users/fergus/Code/platform_one/platformOne/src/platformOne.fsproj.
Finished 'DotNet:restore' in 00:00:09.6192100
Finished 'Install' in 00:00:09.6250540
Starting target 'YarnInstall': NoDescription
  NoDescription
/usr/local/bin/yarn install
yarn install v1.6.0
warning ../../../package.json: No license field
[1/4] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 1.11s.
Finished 'YarnInstall' in 00:00:01.6666980
Starting target 'Watch': NoDescription
  NoDescription
/usr/local/share/dotnet/dotnet  fable  webpack-dev-server --port free
Fable (1.3.15) daemon started on port 51823
CWD: /Users/fergus/Code/platform_one/platformOne
node "/Users/fergus/Code/platform_one/platformOne/node_modules/webpack-dev-server/bin/webpack-dev-server.js"
Bundling for development...
ℹ ｢wds｣: Project is running at http://localhost:8080/
ℹ ｢wds｣: webpack output is served from /
ℹ ｢wds｣: Content not from webpack is served from /Users/fergus/Code/platform_one/platformOne/static/
[WARNING] Cannot find PAKET info for project platformOne.fsproj, it won't be possible to resolve Fable libraries.
[WARNING] Cannot find Fable.Core JS files location for project platformOne.fsproj, using `fable-core`. Add a reference to Fable.Core package or set the fableCore option in the JS client.
Parsing ./src/platformOne.fsproj...
fable: Compiled src/platformOne.fsproj
fable: Compiled src/App.fs
✖ ｢wdm｣: Hash: 573da4340f98e65f0d24
Version: webpack 4.6.0
Time: 9877ms
Built at: 2018-05-02 10:01:16
                                 Asset       Size  Chunks             Chunk Names
  674f50d287a8c48dc19ba404d20fe713.eot    162 KiB          [emitted]
  912ec66d7572ff821749319396470bde.svg    434 KiB          [emitted]
  b06871f281fee6b241d60582ae9369b9.ttf    162 KiB          [emitted]
af7ae505a9eed503f8b8e6982036873e.woff2   75.4 KiB          [emitted]
 fee66e712a8a08eef5805a46892932ad.woff   95.7 KiB          [emitted]
                                app.js    826 KiB     app  [emitted]  app
                              style.js    646 KiB   style  [emitted]  style
                            index.html  571 bytes          [emitted]
Entrypoint app = app.js
Entrypoint style = style.js
[./node_modules/babel-polyfill/lib/index.js] 833 bytes {app} [built]
[./node_modules/core-js/fn/regexp/escape.js] 108 bytes {app} [built]
[./node_modules/core-js/shim.js] 7.99 KiB {app} [built]
[./node_modules/webpack-dev-server/client/index.js?http://localhost:8080] (webpack)-dev-server/client?http://localhost:8080 7.75 KiB {style} {app} [built]
[./node_modules/webpack-dev-server/client/overlay.js] (webpack)-dev-server/client/overlay.js 3.58 KiB {style} {app} [built]
[./node_modules/webpack-dev-server/client/socket.js] (webpack)-dev-server/client/socket.js 1.05 KiB {style} {app} [built]
[./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 489 bytes {style} {app} [built]
[./node_modules/webpack/hot/dev-server.js] (webpack)/hot/dev-server.js 1.66 KiB {style} {app} [built]
[./node_modules/webpack/hot/emitter.js] (webpack)/hot/emitter.js 77 bytes {style} {app} [built]
[./node_modules/webpack/hot/log-apply-result.js] (webpack)/hot/log-apply-result.js 1.31 KiB {style} {app} [built]
[./node_modules/webpack/hot/log.js] (webpack)/hot/log.js 1.03 KiB {style} {app} [built]
[./src/platformOne.fsproj] 25 bytes {app} [built]
[./src/scss/main.scss] 1.19 KiB {style} [built]
[0] multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj 64 bytes {app} [built]
[1] multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./src/scss/main.scss 52 bytes {style} [built]
    + 353 hidden modules

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(15,29): (15,32) error FSHARP: The value, namespace, type or module 'Cmd' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(20,41): (20,44) error FSHARP: The value, namespace, type or module 'Cmd' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(23,4): (23,8) error FSHARP: The value, namespace, type or module 'Hero' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(3,5): (3,11) error FSHARP: The namespace or module 'Elmish' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(4,5): (4,10) error FSHARP: The namespace or module 'Fable' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(46,5): (46,11) error FSHARP: The namespace or module 'Elmish' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(47,5): (47,11) error FSHARP: The namespace or module 'Elmish' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(48,5): (48,11) error FSHARP: The namespace or module 'Elmish' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(5,5): (5,10) error FSHARP: The namespace or module 'Fable' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(50,0): (50,7) error FSHARP: The value, namespace, type or module 'Program' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(52,3): (52,10) error FSHARP: The value, namespace, type or module 'Program' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(54,3): (54,10) error FSHARP: The value, namespace, type or module 'Program' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(56,3): (56,10) error FSHARP: The value, namespace, type or module 'Program' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(58,3): (58,10) error FSHARP: The value, namespace, type or module 'Program' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(6,5): (6,10) error FSHARP: The namespace or module 'Fulma' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
/Users/fergus/Code/platform_one/platformOne/src/App.fs(7,5): (7,10) error FSHARP: The namespace or module 'Fulma' is not defined.
 @ ./src/platformOne.fsproj 1:0-25 1:0-25
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
Module not found: Error: Can't resolve 'fable-core/Symbol' in '/Users/fergus/Code/platform_one/platformOne/src'
 @ ./src/App.fs 5:0-44 6:0-40 16:9-16 40:0-7 50:9-16 72:0-7
 @ ./src/platformOne.fsproj
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj

ERROR in ./src/App.fs
Module not found: Error: Can't resolve 'fable-core/Util' in '/Users/fergus/Code/platform_one/platformOne/src'
 @ ./src/App.fs 7:0-87 29:13-26 34:13-27 61:57-63 66:13-26
 @ ./src/platformOne.fsproj
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server babel-polyfill ./src/platformOne.fsproj
Child html-webpack-plugin for "index.html":
     1 asset
    Entrypoint undefined = index.html
    [./node_modules/html-webpack-plugin/lib/loader.js!./src/index.html] 677 bytes {0} [built]
    [./node_modules/lodash/lodash.js] 527 KiB {0} [built]
    [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 489 bytes {0} [built]
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 497 bytes {0} [built]
ℹ ｢wdm｣: Failed to compile.
```

**Comment by MangelMaxime** [comment]
Can you try on a repo with a path without `_` in it ? This should not be a problem but we never know..

**Comment by MangelMaxime** [comment]
You can also try to delete the obj/bin folder (the script should already do that for you) just to see if it's a caching problem

**Comment by fergusmeiklejohn** [comment]
I tried with no`_`
There is no bin folder either in obj/ or src/obj

Same build failures but different errors depending on whether I'm building the project I was working on yesterday :
```
[WARNING] Cannot find PAKET info for project platformOne.fsproj, it won't be possible to resolve Fable libraries.
[WARNING] Cannot find Fable.Core JS files location for project platformOne.fsproj, using `fable-core`. Add a reference to Fable.Core package or set the fableCore option in the JS client.
```

Or a new project built from the template:
`No executable found matching command "dotnet-fake"`

**Comment by MangelMaxime** [comment]
This is fixed in latest version of the template 1.0.3.

Also, an easier fix than what I proposed before, you can just remove the `.paket/paket.exe.config` file
